src/components/NavigatorSearch/queries/queries.ts

```diff
@@ -1,10 +1,15 @@
 import { gql } from "@apollo/client";
 
-export const checkIfOrderExists = gql`
-  query CheckIfOrderExists($id: ID!) {
-    order(id: $id) {
-      id
-      status
+export const searchOrdersByNumber = gql`
+  query SearchOrdersByNumber($first: Int!, $query: [String!]) {
```

Comment:
suggestion: `$query` suggest that variable contains words, maybe name it like this `orderIds`

Reply:
It's technically a search and all modes in Navigator use `query` so for the sake of consistency I'm keeping `query`